atmo/templates/atmo/jobs/mails/failed_run_alert.mail

```diff
@@ -9,6 +9,8 @@
 {% block body %}
 Your scheduled Spark job "{{ alert.run.spark_job.identifier }}" has failed
 at approximately {{ alert.run.created_at }} UTC.
+Description: {{ alert.run.spark_job.description }}
+
```

Comment:
Can you move this newline above the `Description: ...` so it's separate from the sentence above and only has one line separating the line below?